shieldlib/src/shield_banner.ts

```diff
@@ -133,7 +133,7 @@ function drawBannerTextComponent(
   textComponent: boolean
 ): void {
   const bannerPadding = {
-    top: r.options.bannerPadding,
+    top: 0,
```

Comment:
Removed because the padding is accomplished by placing a gap between successive layouts, not by buffering the layout itself.